docker-compose.ytt.yaml

```diff
@@ -135,6 +135,29 @@ services:
     volumes:
       - db-data:/var/lib/postgresql/data
       - ./postgres/init:/docker-entrypoint-initdb.d/
+  
+  virtuoso:
+    image: tenforce/virtuoso:latest
+    environment: 
+      SPARQL_UPDATE: "true"
+      VIRT_$Database_DatabaseFile: /data/virtuoso_db/virtuoso.db
```

Comment:
Are you sure that the env variables are supposed to contain `$`? 🤔 
From my understanding they should not:
> All properties defined in virtuoso.ini can be configured via the environment variables. The environment variable should be prefixed with VIRT_ and have a format like VIRT_$SECTION_$KEY. $SECTION and $KEY are case sensitive. They should be CamelCased as in virtuoso.ini. E.g. property ErrorLogFile in the Database section should be configured as VIRT_Database_ErrorLogFile=error.log.

https://hub.docker.com/r/tenforce/virtuoso

Reply:
Thanks! I changed it